routes/user.js

```diff
@@ -7,7 +7,7 @@ const router = Router();
 const controller = require("../controllers/user");
 
 // Routes
-
+router.post("/signup",controller.signUp);
```

Comment:
```suggestion
router.post("/",controller.signUp);
```
We need to remove the `signUp` keyword. `POST` request is indicating the resource (users) you are requesting to is going to be created. So no need of it.